app/client/src/features/events/LiveFeedback/LiveFeedbackList.tsx

```diff
@@ -51,15 +66,21 @@ export function LiveFeedbackList({ className, style, fragmentRef }: Props) {
             <div className={classes.content}>
                 <Grid container>
                     <Grid item xs={12}>
+                        <ListFilter
+                            className={classes.listFilter}
+                            onFilterChange={handleFilterChange}
+                            onSearch={handleSearch}
+                            length={filteredList.length}
+                        />
                         <List disablePadding>
                             {displayLiveFeedback ? (
-                                liveFeedback.map((feedback) => (
+                                liveFeedback.filter(feedback => filteredList.includes(feedback)).map((feedback) => (
```

Comment:
Hmm, this seems somewhat expensive?  Could you explain what happens if you don't have this?  It seems like feedback that doesn't include actual text is being filtered out, but is there a use case where feedback wouldn't have text?  If so, maybe you could memoize this to save on performance?

Reply:
Without the filter, it still shows all the feedback cards instead of the ones that match the search filter. I'll try to memoize it.

Reply:
I just realized, after looking at how the question list filter was implemented, that I could directly map the filtered list instead so I'll go ahead and change it.